### PR TITLE
Vis and project index page boxes are now more consistent

### PR DESCRIPTION
**Pull request by stowellm**
Addresses #2059
:cake:   :goat:

### CODE REVIEW

File: app/views/visualizations/index.html.erb [hunk 1]
```diff
@@ -1,34 +1,33 @@
 <div class="row">
   <div class="col-lg-12">
     <div class="search-box">
-      <%=form_tag visualizations_path, method: 'get', id: 'visualizations_search' do%>
-
+      <%=form_tag visualizations_path, method: "get",
```
Comment (jaypoulz) [inline app/views/visualizations/index.html.erb]: spacing around <%=

File: app/views/shared/_vis_box.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+     </a>
+     <div class="item-desc">
+       <div class="item-title">
+         <%= link_to @box_vis.name.html_safe, @box_vis %>
+         <% if @box_vis.featured %>
+           <span> (featured)</span>
+         <% end %>
+       </div>
+       by <a href="<%=user_url(@box_vis.owner)%>">
+          <%=@box_vis.owner.name%> </a><br/>
```
Comment (jaypoulz) [inline app/views/shared/_vis_box.html.erb]: ditto

File: app/views/shared/_project_box.html.erb [hunk 3]
```diff
@@ -17,8 +17,8 @@
           <span> (featured) </span>
         <% end %>
       </div>
-      by <%= @box_project.owner.name %>
-      <br/>
+      by <a href="<%=user_url(@box_project.owner)%>">
+         <%=@box_project.owner.name%> </a><br/>
```
Comment (jaypoulz) [inline app/views/shared/_project_box.html.erb]: ditto

File: app/views/projects/index.html.erb [hunk 4]
```diff
@@ -1,85 +1,94 @@
...
+                <%= check_box_tag "has_data", "yes", @params["has_data"] %>
                 Has Data
               </label>
             </div>
           </div>
         </div>
-          
-        <div class='clear'></div>
+        <div class="clear"></div>
       <%end%>
```
Comment (jaypoulz) [inline app/views/projects/index.html.erb]: ditto

File: app/views/projects/index.html.erb [hunk 5]
```diff
@@ -1,85 +1,94 @@
 <div class="row">
   <div class="col-lg-12">
     <div class="search-box">
-      <%=form_tag projects_path, method: 'get', id: 'projects_search' do%>
+      <%=form_tag projects_path, method: "get", id: "projects_search" do%>
```
Comment (jaypoulz) [inline app/views/projects/index.html.erb]: ditto

**Comment by stowellm** [comment]
@jaypoulz Ready for re-review.

**Pull request merged**